packages/main/src/Menu.ts

```diff
@@ -180,6 +183,30 @@ class Menu extends UI5Element {
 	@property({ type: Boolean })
 	open!:boolean;
 
+	/**
+	 * Defines if a loading indicator would be displayed inside the corresponding ui5-menu popover.
+	 *
+	 * @name sap.ui.webc.main.List.prototype.busy
+	 * @type {boolean}
+	 * @defaultvalue false
+	 * @public
+	 * @since 1.13.0
+	 */
+	@property({ type: Boolean })
+	busy!: boolean;
+
+	/**
+	 * Defines the delay in milliseconds, after which the busy indicator will displayed inside the corresponding ui5-menu popover.
```

Comment:
Defines the delay in milliseconds, after which the busy indicator will be displayed inside the corresponding ui5-menu popover.

Reply:
done